### ISSUE TITLE
[Regression] Can't add games

### ISSUE THREAD

**Issue by deusnovus** [RPCS3/rpcs3#14519]
### Quick summary

**The problem:**

I can't seem to be able to add games to my home list anymore. I demonstrate the problem in the video below:

[cantaddgames.webm](https://github.com/RPCS3/rpcs3/assets/32219080/0431c7b1-6b3a-4d51-af18-460680426339)

**Things to note:**

1. I used to be able to add games perfectly fine.
2. I can still play games by clicking Open and choosing the game directory.
3. When in the Add Games window, I tried both going inside the folder (like in the video) and choosing the folder in the parent directory.

**My system info:**

Dell Inspiron 16 Plus 7610
CPU: 11th Gen Intel® Core™ i7-11800H
RAM: 32GB
GPU1: NVIDIA GeForce RTX™ 3050 Ti Laptop GPU
GPU2: Intel® UHD Graphics (TGL GT1)
OS: Fedora 38 / GNOME 44.3 / Wayland

### Build with regression

v0.0.29-15504-eb978a74 Alpha | master

### Attach two log files

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/12388313/RPCS3.log)

**Comment by FlexBy420** [comment]
You need to choose parent folder of games, not the game folder itself.

**Comment by MSuih** [comment]
Did you accidentally hide the game list? Try `View -> Show game list`

**Comment by deusnovus** [comment]
> You need to choose parent folder of games, not the game folder itself.

Yes, as I mentioned above, I tried that as well, but doesn't do anything.

**Comment by deusnovus** [comment]
> Did you accidentally hide the game list? Try `View -> Show game list`

Oh, it seems that was part of the problem. But I still can't add any newer games after a recent update. Let me demonstrate the problem with another video:

[cantaddgames2.webm](https://github.com/RPCS3/rpcs3/assets/32219080/a9ee7f72-65b5-4d8b-9421-b2741950d8cd)

**Comment by MSuih** [comment]
Additional things you can try:

* Make sure you haven't accidentally hidden any games: `View -> Show hidden entries`
* Test that the games haven't somehow gotten converted to wrong catefory with `View -> Game categories` and by enabling everything there.

**Comment by deusnovus** [comment]
> Additional things you can try:
> 
>     * Make sure you haven't accidentally hidden any games: `View -> Show hidden entries`
> 
>     * Test that the games haven't somehow gotten converted to wrong catefory with `View -> Game categories` and by enabling everything there.

I tried the above, but I still don't have any other games apart from this existing one and its save file.

I also noticed something interesting just now: when I click `View -> Show game list`, close and relaunch the appimage, the list becomes hidden yet again by default, just like in the first video. That didn't used to be the case.

**Comment by MaroonMike1** [comment]
I am having the same issue trying to add Resistance - Fall of Man (BCUS98107).  The error box I get is -
"Nothing to add! - Could not find any new software".

I am able to run it under File -> Boot Game. but that does not allow a custom configuration.

**Comment by egjream** [comment]
Did you guys figure it out, I have the same issue... What's going on?

**Comment by MaciejReszotnik** [comment]
@egjream 

On my SteamDeck the issue came up due to folder set up by EmuDeck not having sufficient permissions. Either change the folder location or change folder permissions with chmod terminal command.

**Comment by gillopillo** [comment]
I have the same issue. Cant add games to the newer versions.
I had an old version installed in another folder. I added the games there without a problem from the same exact folder.
then I updated this Rpcs3 to the latest version from the GUI and lost the add games functionality again. No matter what I do, game folders that used to be added without an issue now give me that error.

**Comment by chihabeddineAourinmouche** [comment]
> I am having the same issue trying to add Resistance - Fall of Man (BCUS98107). The error box I get is - "Nothing to add! - Could not find any new software".
> 
> I am able to run it under File -> Boot Game. but that does not allow a custom configuration.

RPCS3 will not accept ISO formats. The solution is to:
- Extract the content of the ISO into a folder (With a name indicating the game)
- Copy that folder into a parent folder that will contain the games you would like to add
- Then add game like you would normally do

This worked for me.

**Comment by AniLeo** [comment]
Removed all the unrelated comments. The issue here is insufficient permissions. We need to add a clear error message for missing permissions and close this.

**Comment by RealSentientPotato** [comment]
issue has worsened

**Comment by RealSentientPotato** [comment]
The issue now appears to be worse. multiple games i had ripped from my library for rpcs3 which had been working now no longer show up

**Comment by AniLeo** [comment]
We already know the root cause for the bug in the OP. I'm unfortunately having to lock this issue due to many unrelated reports of different issues with the same symptoms and just reports from straight up pirated shit.
